src/database/queries/addData.js

```diff
@@ -0,0 +1,22 @@
+const connection = require('../config/db_connection');
+
+const addUser = (userInfo) => {
+  const queryObj = {
```

Comment:
@AhmedAl-Almi Looks good!  You could use `hashPass` to hash the password right in this function, once #72 is merged.  I see that `hashPass` returns a promise so this function then will have to change slightly, to wait for the promise.  This looks great for now!